fern/api-reference/hyperliquid/hyperliquid-api-faq.mdx

```diff
@@ -2,7 +2,6 @@
 title: Hyperevm API FAQ
 description: Frequently asked questions about the Hyperevm API
 subtitle: Frequently asked questions about the Hyperevm API
```

Comment:
☝️  maybe a couple missed hyperevm -> hyperliquids

Reply:
I only updated slugs and filenames, not any content. Doing that is a bit more complicated because some of the content is about HyperEVM the execution layer and some is about hyperliquid the blockchain. Ideally this is updated by content team separately.